soccer/SystemState.hpp

```diff
@@ -70,9 +70,14 @@ class SystemState {
      */
 
     /** @ingroup drawing_functions */
-    void drawLine(const Geometry2d::Line& line, const QColor& color = Qt::black,
+    void drawLine(const Geometry2d::Segment& line,
```

Comment:
Hmm, if we have both a drawLine and a drawSegment method, we should probably make them do what the name says.  You're right though that we almost always want to draw a segment.  Can we change the calls to drawLine to be drawSegment?